### PR TITLE
[Feature Store] Remove id from spark_stats

### PR DESCRIPTION
**Pull request by speretz-ig**
[ML-1272](https://jira.iguazeng.com/browse/ML-1272)

### CODE REVIEW

**Review by gtopper — COMMENTED** [review]
Great, just need to add test coverage. @yaronha, any reason that this column was added originally?

File: tests/system/feature_store/test_spark_engine.py [hunk 1]
```diff
@@ -138,6 +138,96 @@ def test_basic_remote_spark_ingest(self):
...
+                "movements": 10.0,
+                "patient_id": "838-21-8151",
+                "room": 2.0,
+                "rr": 25.0,
+                "spo2": 99.0,
+                "turn_count": 3.0,
+            },
+        }
+        equal_Df = pd.DataFrame(expexted_result)
+        assert stats.drop("hist", axis=1).equals(equal_Df)
```
Comment (gtopper) [inline tests/system/feature_store/test_spark_engine.py:229]:
```suggestion
        expected_df = pd.DataFrame(expexted_result)
        assert stats_df.drop("hist", axis=1).equals(expected_df)
```

**Comment by gtopper** [comment]
Please don't force push.

File: tests/system/feature_store/test_spark_engine.py (hunk 1 above)
Comment (speretz-ig) [reply tests/system/feature_store/test_spark_engine.py]: Done

**Review by gtopper — APPROVED** [review]
✅

**Pull request merged**